v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -916,6 +918,77 @@ func init() {
 
 			return buf.String(), nil
 		}),
+		"resolve": makeDslWithOptionalArgsFunction("(host string, format string) string", func(args ...interface{}) (interface{}, error) {
```

Comment:
`makeDslWithOptionalArgsFunction` doesn't seem appropriate here. I think `makeMultiSignatureDslFunction` would make more sense, as it looks like you can have either one or two arguments:
```
(host string, format string)
(host string)
```
If the arguments are more than two, we should return `invalidDslFunctionError` (similarly to the `split` function)